src/client/pages/OfferNew/Introduction/index.tsx

```diff
@@ -21,16 +23,20 @@ const HERO_HEIGHT = '400px'
 
 const Hero = styled.div`
   width: 100%;
-  height: ${HERO_HEIGHT};
   display: flex;
   flex-direction: column;
   align-items: center;
+  background-color: ${colorsV3.gray100};
+
+  ${LARGE_SCREEN_MEDIA_QUERY} {
+    height: ${HERO_HEIGHT};
+  }
 `
 
 const HeroImageWrapper = styled.div`
   width: 100vw;
   height: ${HERO_HEIGHT};
-  background: ${colorsV3.gray900};
+  background-color: ${colorsV3.gray900};
```

Comment:
What's the point of using the `background-color` property instead of the shorthand `background`? Also, we do use `background` for setting the background color in like 60 places in the repo.

Reply:
Good question! 😃 This is a force of habit for me, since the shorthand sets a few other properties to it's default values if you don't specify them, which can cause unwanted behaviour. In most cases it will be fine, but I have found myself having to change this in previous projects when finding bugs, so I try to stick to setting the specific value :) This is of course just an opinion. Being the nerd that I am about this I think this post points out the pros and cons 🤓
https://csswizardry.com/2016/12/css-shorthand-syntax-considered-an-anti-pattern/